Utils/ContextToYML.yml

```diff
@@ -0,0 +1,202 @@
+commonfields:
+  id: ContextToYML
+  version: 2
+name: ContextToYML
+script: |
+  import yaml
+
+  """
+  This script parse a Context output into yml formatted file with the context path of each field.
+  For example:
+  {
+      "id":12131,
```

Comment:
remove similar paths in example to make it simpler. you can use `...` to indicate more of the same.
```
{
     "id": 1234,
      "summary":"do-not-delete",
      ...
      "issuetype": {
          "id":10004,
          "name":"Bug"
      },
      "status": {
          "id":10003,
          "StatusCategory": {
                  "key":"new"
          },
          ...
      }
  }
}
```